casper/src/main/scala/coop/rchain/casper/util/comm/DeployRuntime.scala

```diff
@@ -13,12 +11,17 @@ import scala.util.{Failure, Success, Try}
 
 object DeployRuntime {
 
-  def propose[F[_]: DeployService: Monad](): F[Unit] =
+  def propose[F[_]: Monad: Capture: DeployService](): F[Unit] =
     DeployService[F].createBlock().flatMap {
       case Some(block) =>
-        DeployService[F].addBlock(block)
+        println(s"Response: Successfully created block")
```

Comment:
Please include this under a `Capture` in the for-comprehension as well.

Reply:
missed it, will do.